hasjob/templates/reject_domain_email.html

```diff
@@ -0,0 +1,15 @@
+{% extends "inc/email_layout_lite.html" %}
+
+{% block content %}
+  <p>Hi {{ post.company_name }},</p>
+  <p>This is regarding your job post on Hasjob titled "<strong>{{ post.headline }}</strong>". Unfortunately, your post is in violation of our <a href="{{ url_for('terms_of_service', _external=true) }}">Terms of Service</a> and has been <strong>banned</strong>. As a result all the job posts under the same domain has been taken down and your user account has been banned too. Your post was reviewed by {{ post.reviewer.fullname }}, who left the following comment:</p>
```

Comment:
Since banning is more severe than rejecting one post, this warrants different text:

```
This is regarding your job post(s) on Hasjob. A moderator ({{ post.reviewer.fullname }}) reviewed your post(s) and determined that you represent a recruitment agency or other class of employer that is not accepted on Hasjob, and has therefore taken down the following jobs:

(list here)

The moderator provided the following reason: (message)

If you believe this is incorrect and would like to appeal, please forward this email to support@hasgeek.com with an explanation.
```